Prevents dead or unconscious people from removing handcuffs.

Ruzihm opened the pull request:
### Purpose
Fixes #2993

### Notes:
Problem was caused by a reference to `PlayerHealth.serverPlayerConscious` which doesn't seem to actually do anything.

Ruzihm committed:
Prevents dead or unconscious people from removing handcuffs.

DooblyNoobly commented on UnityProject/Assets/Scripts/Clothing/RestraintOverlay.cs:
```diff
@@ -98,9 +98,11 @@ IEnumerator UncuffCountDown(CancellationToken cancelToken)
 	bool CanUncuff()
 	{
-		if (!thisPlayerScript.playerHealth.serverPlayerConscious ||
+		PlayerHealth playerHealth = thisPlayerScript.playerHealth;
```
Pretty sure PlayerHealth is null if you are a ghost. Is there anyway CanUncuff() might get called if you are a ghost? Maybe you should check that playerHealth exists and if it doesn't then return false. To be safe.

Ruzihm replied:
Great point. `playerHealth` was already being dereferenced before this pull request but now is as good a time as any to have that check.

Ruzihm committed:
Addresses this concern:

> Pretty sure PlayerHealth is null if you are a ghost. Is there anyway CanUncuff() might get called if you are a ghost? Maybe you should check that playerHealth exists and if it doesn't then return false. To be safe.
